index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./Style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <nav class="navbar">
+                <li class="logo"><img src="./images/My Logo.png" alt="none"></li>
+                <li class="union"><img src="./images/Union.png" alt="none"></li>
+            </nav>
+        </header>
+        <section class="headline">
+            <h1>
+                I’m Shakir <br>
+                Glad to see you!
+            </h1>
+            <p class="body-text">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <li class="connect">
+                Let's connect
+            </li>
+            <ul class="socials">
+                <li class="icons">
+                    <a href="">
+                        <img src="./images/Vector-1.png" alt="none">
+                    </a>
+                </li>
+                <li class="icons">
+                    <a href="">
+                        <img src="./images/Linkedin icon.png" alt="none">
+                    </a>
+                </li>
+                <li class="icons">
+                    <a href="">
+                        <img src="./images/Vector.png" alt="none">
+                    </a>
+                </li>
+                <li class="icons">
+                    <a href="">
+                        <img src="./images/Vector.svg" alt="none">
+                    </a>
+                </li>
+            </ul>
+        </section>
+    </div>
+</body>
```

Comment:
Also here,  Kindly add the background images as requested by the previous reviewers so that your work can be in line with [Figma]😊(https://www.figma.com/file/b3dHWtiz1jvo4HUemk4dZP/Microverse-Student-Project-1-(Copy)?node-id=0%3A1&t=FySRMNXZc4fsu2Cy-0) design.

| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="326" alt="image" src="https://user-images.githubusercontent.com/73854999/205115099-a99be9ef-acdd-4567-af61-1394e2fafd8e.png">| <img width="379" alt="image" src="https://user-images.githubusercontent.com/73854999/205114948-814405a3-97e7-417d-a990-872e605bab0f.png">|

Reply:
Hi @Lizdev-05 , the below snapshot shows the code at my end. And, it works perfectly. I think it's an issue of "." missing before the image URL when you're trying to run the code. Can you check it with a "." in front of URL in style.css.
![my-test](https://user-images.githubusercontent.com/94363151/205119157-1cc7e6cd-af20-44d5-b35e-d3b9ef945443.PNG)